incubator/couchdb/templates/statefulset.yaml

```diff
@@ -87,12 +97,14 @@ spec:
 {{ toYaml .Values.nodeSelector | indent 8 }}
 {{- end }}
       volumes:
-        - name: config
+        - name: init-config
           configMap:
             name: {{ template "couchdb.fullname" . }}
             items:
               - key: inifile
                 path: chart.ini
+        - name: config
+          emptyDir: {}
```

Comment:
Should you now add a persistent volume to ensure config sticks around as the container moves through the cluster?